addons/survey/tests/test_survey_ui_feedback.py

```diff
@@ -179,12 +179,17 @@ def test_04_public_survey_with_triggers(self):
             'access_token': '3cfadce3-3f7e-41da-920d-10fa0eb19527',
             'access_mode': 'public',
             'users_can_go_back': True,
-            'questions_layout': 'one_page',
+            'questions_layout': 'page_per_section',
```

Comment:
It's a pity if it does proceed with submitting without doing local validation first if `questions_layout` is "one_page". Shouldn't we try and fix this (so this test can stay a bit simpler)? Unless this is not possible because of "can_go_back" ?